src/js/components/WorldMap/__tests__/WorldMap-test.tsx

```diff
@@ -69,6 +69,7 @@ describe('WorldMap', () => {
         <WorldMap
           places={[
             {
+              // @ts-ignore
               'data-testid': 'Sydney',
```

Comment:
Is `data-test-id` reflected in the test snapshots at all or can we remove it entirely? Would prefer removal over `// @ts-ignore` if it's not affecting the test output at all and it's just left from some legacy code.

Reply:
It is in the snapshots, and I think it's needed to select the target for the events. Tests break without it

Reply:
'data-testid' is being used in the query statements for the tests but I am thinking we can remove it and change the query statements. For example the following query uses 'data-testid':
`fireEvent.mouseOver(getByTestId('Sydney'));`
I think we could change it to something like this:
`fireEvent.mouseOver(screen.getByRole('button', {name: /Sydney/i}));`